File: src/main/java/com/elchologamer/userlogin/util/extensions/QuickMap.java
package com.elchologamer.userlogin.util.extensions;

import java.util.HashMap;

public class QuickMap<K, V> extends HashMap<K, V> {

    public QuickMap() {
    }

    public QuickMap(K firstKey, V firstValue) {
        this.put(firstKey, firstValue);
    }

    public QuickMap<K, V> set(K key, V value) {
        super.put(key, value);
        return this;
    }
}

File: src/main/kotlin/com/elchologamer/userlogin/util/QuickMap.kt
package com.elchologamer.userlogin.util

class QuickMap<K, V>(firstKey: K, firstValue: V) : HashMap<K, V>() {
    init {
        put(firstKey, firstValue)
    }

     operator fun set(key: K, value: V): QuickMap<K, V> {
        super.put(key, value)
        return this
    }
}